src/estimation/src/BerdyHelper.cpp

```diff
@@ -542,6 +543,11 @@ IndexRange BerdyHelper::getRangeSensorVariable(const SensorType type, const unsi
         sensorOffset += 3*m_sensors.getNrOfSensors(THREE_AXIS_ANGULAR_ACCELEROMETER);
     }
 
+    if( type > THREE_AXIS_FORCE_TORQUE_CONTACT )
```

Comment:
Sorry.. I probably saw this method a lot of time, but only now I am wondering.. why `type > XXX` ?

Reply:
Basically is quite a crude way to express the logic "if the sensor comes after all the sensors of type XXX in serialization, then you have to use the proper offset when computing the sensor range in the sensor serialization.